tests/unit/services/test_lifecycle.py

```diff
@@ -825,8 +825,16 @@ def test_no_builds_error(fake_parts_lifecycle):
 @pytest.mark.parametrize("fake_build_plan", [2, 3, 4], indirect=True)
 def test_multiple_builds_error(fake_parts_lifecycle):
     """Build plan contains more than 1 item."""
-    with pytest.raises(errors.MultipleBuildsError):
+    with pytest.raises(errors.MultipleBuildsError) as e:
         fake_parts_lifecycle.run("prime")
+    assert str(e.value) == (
+        "Multiple builds match the current platform: "
```

Comment:
can we test against the actual string here? I'd like to see how the end result looks

Reply:
Agreed

Reply:
@tigarmo @lengau How's this look? [38312b9](https://github.com/canonical/craft-application/pull/428/commits/38312b981aa43eb9dda9eca39194593ec5df2c6f)